code.json

```
@@ -0,0 +1,87 @@
+{
+  "name": "bluebutton-web-server",
+  "description": "Blue Button API",
+  "longDescription": "This server serves as a data provider for sharing Medicare claims data with third parties. The server connects to Medicare.gov for authentication, and uses OAuth2 to confirm permission grants to external app developers. The data itself comes from a back end FHIR server (https://github.com/CMSgov/bluebutton-data-server), which in turn pulls data from the CMS Chronic Conditions Warehouse (https://www.ccwdata.org)",
```

The backend fhir server has changed, so just need to point to the updated repo for that.
```suggestion
  "longDescription": "This server serves as a data provider for sharing Medicare claims data with third parties. The server connects to Medicare.gov for authentication, and uses OAuth2 to confirm permission grants to external app developers. The data itself comes from a back end FHIR server (https://github.com/CMSgov/beneficiary-fhir-data), which in turn pulls data from the CMS Chronic Conditions Warehouse (https://www.ccwdata.org)",
```